extensions/amp-subscriptions/0.1/entitlement.js

```diff
@@ -64,7 +64,7 @@ export class Entitlement {
     this.subscriptionToken = subscriptionToken;
     /** @const {boolean} */
     this.loggedIn = loggedIn;
-    /** @const {?MeteringData} */
```

Comment:
Add `@type` instead then. But why is it not `@const` anymore?

Reply:
Done